src/dstore/pmix_esh.c

```diff
@@ -530,7 +530,7 @@ int _esh_init(pmix_info_t info[], size_t ninfo)
 {
     int rc;
     size_t n;
-    char *tmp;
+    char *tmp_path;
     size_t tbl_idx;
     struct stat st = {0};
     ns_map_data_t *ns_map = NULL;
```

Comment:
`if (PMIX_PROC_SERVER == pmix_globals.proc_type) {` below - should we use `is_server()` instead?